src/yarp.c

```diff
@@ -2037,6 +2037,65 @@ parse_statements(yp_parser_t *parser, yp_context_t context) {
   return statements;
 }
 
+static void
+parse_hash_assocs(yp_parser_t *parser, yp_node_t *hash) {
+  while (parser->current.type != YP_TOKEN_BRACE_RIGHT && parser->current.type != YP_TOKEN_EOF) {
+    while (accept(parser, YP_TOKEN_NEWLINE));
+
+    if (hash->as.hash_node.elements.size != 0) {
+      expect(parser, YP_TOKEN_COMMA, "expected a comma between hash entries.");
+    }
+
+    while (accept(parser, YP_TOKEN_NEWLINE));
+    yp_node_t *element;
+
+    switch (parser->current.type) {
+    case YP_TOKEN_STAR_STAR: {
```

Comment:
`case`s are usually indented one level below the `switch`, perhaps some C linter would catch that eventually, but just mentioning as it caught my eye.

Reply:
Thanks! I think I gotta change an editor config. 🤦‍♂️